slices/git.yaml

```diff
@@ -0,0 +1,34 @@
+package: git
+
+essential:
+  - git_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libpcre2-8-0_libs
+      - zlib1g_libs
+    contents:
+      /usr/bin/git:
+      /usr/bin/git-shell:
+      /usr/bin/scalar:
+
+  core:
+    essential:
+      - git_bins
+      - libcurl3t64-gnutls_libs
+      - libexpat1_libs
+    contents:
+      /usr/lib/git-core/**:
+
+  perl5:
+    essential:
+      - git_core
+    contents:
+      /usr/share/perl5/Git.pm:
+      /usr/share/perl5/Git/*:
```

Comment:
I think this is also extra, that we currently don't need. I have no strong objections against having it. But the problem would be removing it in future if we add it now.

Reply:
I agree. done!